### ISSUE TITLE
GreatBarrierReef Analysis

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#146]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : GreatBarrierReef Analysis
:red_circle: **Aim** : Analyze the data from the Great Barrier reef.
:red_circle: **Dataset** : https://www.kaggle.com/andradaolteanu/2021-greatbarrierreef-prep-data
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

Hello, ML-Crate contributors, this issue is only for the contribution purposes and allocated only to the participants of SWOC 2.0 Open Source Program and JWOC '22 Open Source Program.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.
- This issue is only for 'SWOC' and 'JWOC' contributors of 'ML-Crate' project.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID :
- Approach for this Project :
- What is your participant role?
    - [ ] SWOC 2.0 Participant.
    - [ ] JWOC 2022 Participant.
    - [ ] Contributor

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Nirvik07** [comment]
@abhisheks008 in this project do we have to do analysis with only train.csv or anything else?

**Comment by abhisheks008** [comment]
You need to use deep learning methods to create a model which will determine whether a particular area is a part of Great Barrier Reef or, not.

**Comment by why-aditi** [comment]
Aditi Kala
Github:- https://github.com/why-aditi
Participation ID:- NA
Approach: Exploratory Data Analysis (EDA) to understand the dataset's structure, identify any anomalies, and visualize key variables such as water quality parameters and species diversity. Following EDA, data preprocessing will be essential to handle missing values, encode categorical variables, and scale numerical features as needed. Next, the project will implement and compare the performance of some machine learning algorithms. Model evaluation will focus on metrics like accuracy to determine the best-performing algorithm. Optionally, fine-tuning via hyperparameter optimization can further enhance model performance.
Participation Role:- SSOC Season 3

**Comment by abhisheks008** [comment]
> Aditi Kala Github:- https://github.com/why-aditi Participation ID:- NA Approach: Exploratory Data Analysis (EDA) to understand the dataset's structure, identify any anomalies, and visualize key variables such as water quality parameters and species diversity. Following EDA, data preprocessing will be essential to handle missing values, encode categorical variables, and scale numerical features as needed. Next, the project will implement and compare the performance of some machine learning algorithms. Model evaluation will focus on metrics like accuracy to determine the best-performing algorithm. Optionally, fine-tuning via hyperparameter optimization can further enhance model performance. Participation Role:- SSOC Season 3

One issue at a time.

**Comment by Sanjanah8** [comment]
Can u assign me to this

**Comment by abhisheks008** [comment]
> Can u assign me to this

Share your approach please.
